a1/Makefile

```diff
@@ -2,7 +2,7 @@
 all: test
 
 clean:
-	find . -type f \( -iname \*.o -o -iname \*.cmx \) | xargs rm
+	ocamlbuild -clean

```

Comment:
Hey @will-wow 

On your machine do you see that the `ocamlbuild -clean` command removes build artifacts?

Here's what happens on mine:

```
✔ ~/dev/cos-326-fall-2017/a1 [laser|✚ 1]
17:28 $ ls
Makefile a1.ml    test     test.ml
✔ ~/dev/cos-326-fall-2017/a1 [laser|✚ 1]
17:28 $ make
find . -type f \( -iname \*.cache -o -iname \*.log -o -iname \*.cm\* -o -iname \*.o \) | xargs rm
ocamlfind ocamlopt -o test -package oUnit -linkpkg -g a1.ml test.ml
./test
............
monte_pi answer: 3.14

Ran: 12 tests in: 0.11 seconds.
OK
✔ ~/dev/cos-326-fall-2017/a1 [laser|✚ 1]
17:28 $ ls
Makefile                                 oUnit-A1-Lasers-MacBook-Pro.local#02.log
a1.cmi                                   oUnit-A1.cache
a1.cmx                                   test
a1.ml                                    test.cmi
a1.o                                     test.cmx
oUnit-A1-Lasers-MacBook-Pro.local#00.log test.ml
oUnit-A1-Lasers-MacBook-Pro.local#01.log test.o
✔ ~/dev/cos-326-fall-2017/a1 [laser|✚ 1]
17:28 $ ocamlbuild -clean
Finished, 0 targets (0 cached) in 00:00:00.
00:00:00 0    (0   ) STARTING                                                            -------- |✔ ~/dev/cos-326-fall-2017/a1 [laser|✚ 1]
17:28 $ ls
Makefile                                 oUnit-A1-Lasers-MacBook-Pro.local#02.log
a1.cmi                                   oUnit-A1.cache
a1.cmx                                   test
a1.ml                                    test.cmi
a1.o                                     test.cmx
oUnit-A1-Lasers-MacBook-Pro.local#00.log test.ml
oUnit-A1-Lasers-MacBook-Pro.local#01.log test.o
```

Reply:
Ah that's a good point - I guess `ocamlbuild -clean` actually only removes the `_build` directory. I think I've got a solve though - it turns out we can use `ocamlbuild` for running our tests, and then it puts all the build artifacts where they should be, in the `_build` directory.